components/log-viewer-webui/client/src/pages/SearchPage/SearchControls/TimeRangeInput/index.module.css

```
@@ -0,0 +1,15 @@
+.timeRangeInputContainer {
+    display: inline-flex;
+}
+
+/* Makes border flush with range picker */
+.select-container :global(.ant-select-selector) {
```

since this is only applied when the date-picker shows up, how about
```suggestion
.selectWithRightAddon :global(.ant-select-selector) {
```

---

I called it customSelected, since i already have another component that refers to the combined element